ripe/atlas/tools/probes/__init__.py

```diff
@@ -11,10 +11,6 @@ class Probe(object):
 
     EXPIRE_TIME = 60 * 60 * 24 * 30
 
-    def __init__(self, id, meta_data=None):
-
-        return CProbe(id, meta_data)
-
```

Comment:
Anyway, we don't use this feature for now, so it's okay that you delete it. If you need it we can come back to it. I am merging it :)